tests/integration/test_logcollector/test_location/test_location.py

```diff
@@ -0,0 +1,246 @@
+         'log_format': 'syslog', 'file_type': 'wildcard_file'},
+        {'location': '/tmp/wazuh-testing/c*test.txt',
+         'files': ['/tmp/wazuh-testing/c1test.txt', '/tmp/wazuh-testing/c2test.txt',
+                   '/tmp/wazuh-testing/c3test.txt'], 'log_format': 'syslog',
+         'file_type': 'wildcard_file'},
+        {'location': '/tmp/wazuh-testing/duplicated/duplicated.txt',
+         'files': ['/tmp/wazuh-testing/duplicated/duplicated.txt'],
+         'log_format': 'syslog', 'path_2': '/tmp/wazuh-testing/duplicated/duplicated.txt',
+         'file_type': 'duplicated_file'},
+        {'location': '/tmp/wazuh-testing/file.log-%Y-%m-%d',
+         'files': [f'/tmp/wazuh-testing/file.log-{date}'], 'log_format': 'syslog',
+         'file_type': 'single_file'},
+        {'location': '/tmp/wazuh-testing/multiple-logs/*', 'files': ['/tmp/wazuh-testing/multiple-logs/multiple'],
+         'log_format': 'syslog', 'file_type': 'multiple_logs'}
+    ]
+
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOCATION'], x['LOG_FORMAT']}" for x in parameters]
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Fixtures
+@pytest.fixture(scope="module")
+def create_directory():
+    """Create expected directories."""
+    if sys.platform == 'win32':
+        os.makedirs(fr'{temp_dir}\wazuh-testing\multiple-logs', exist_ok=True)
+        os.makedirs(fr'{temp_dir}\wazuh-testing\depth1\depth2', exist_ok=True)
+        os.makedirs(fr'{temp_dir}\wazuh-testing\duplicated', exist_ok=True)
+    else:
+        os.makedirs('/tmp/wazuh-testing/multiple-logs', exist_ok=True)
+        os.makedirs('/tmp/wazuh-testing/depth1/depth2', exist_ok=True)
+        os.makedirs('/tmp/wazuh-testing/duplicated', exist_ok=True)
+    yield
+
+    if sys.platform == 'win32':
+        rmtree(fr'{temp_dir}\wazuh-testing', ignore_errors = True)
+    else:
+        rmtree('/tmp/wazuh-testing')
+
+
+@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def get_local_internal_options():
+    """Get configurations from the module."""
+    return local_internal_options
+
+
+@pytest.fixture(scope='module')
+def create_files(request, get_configuration):
+    """Create expected files."""
+    files = get_configuration['metadata']['files']
+    file_type = get_configuration['metadata']['file_type']
+
+    for file_location in files:
+        if file_type == 'non_existent_file':
+            pass
+        elif file_type == 'multiple_logs':
+            for i in range(2000):
+                name = f'{file_location}{i}.txt'
+                with open(name, 'w') as file:
+                    file.write(' ')
+        else:
+            with open(file_location, 'w') as file:
+                file.write(' ')

```

Comment:
```suggestion
   if file_type != 'non_existent_file':
      for file_location in files:
         if file_type == 'multiple_logs':
              for i in range(2000):
                  with open(f'{file_location}{i}.txt', 'w') as file:
                      file.write(' ')
          else:
              with open(file_location, 'w') as file:
                  file.write(' ')

```
Just a question. Do you need to write a white space for the test? If not, you can create the file with just:
```python
open(name, 'w').close()
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/5170c4e8209b55c928abc5fcf5d64a9cab223fae